src/index.html

```diff
@@ -6,9 +6,38 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com"
+          crossorigin="#">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo">
+        <img src="./images/logo.png" alt="moyo-logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav-ul">
+          <li class="nav-li"><a class="nav-link is-active" href="#">apple</a></li>
+          <li class="nav-li"><a class="nav-link" href="#">samsung</a></li>
```

Comment:
[Checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md):
- Add empty lines between multiline sibling blocks of HTML
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/d925db3f-058a-4212-9e49-bab45eb32e1c)
- And don't put nested elements in one line with parent element

Reply:
Thanks. Done.